services/github/github-pull-request-check-state.tester.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const t = require('../create-service-tester')()
+module.exports = t
+
+t.create('github pull request check state')
+  .get('/s/pulls/badges/shields/1110.json')
+  .expectJSON({ name: 'checks', value: 'failure' })
+
+t.create('github pull request check state (pull request not found)')
+  .get('/s/pulls/badges/shields/5110.json')
+  .expectJSON({ name: 'checks', value: 'pull request or repo not found' })
+
+t.create('github pull request check contexts')
+  .get('/contexts/pulls/badges/shields/1110.json')
+  .expectJSON({ name: 'checks', value: '1 failure' })
```

Comment:
Could we also add a (mocked) test for the 'commit not found' case.
https://github.com/badges/shields/blob/7a664ca3e845c840fa299fa1ee1ee0e787593038/services/codeclimate/codeclimate.tester.js#L127-L138
is probably a good example to draw on.

Reply:
Added!